app/auth/controllers.py

```diff
@@ -222,3 +224,63 @@ def get_s3_signed_url():
     except Exception as e:
         app.logger.error(e)
         return handle_error('GENERIC_ERROR', e.message, 500)
+
+
+@datastore_blueprint.route('/authorize', methods=['POST'])
```

Comment:
Where is the route we are deleting and replacing with this route (or redirecting to this route if we are maintaining backwards compatibility)?